luigi/execution_summary.py

```diff
@@ -30,36 +30,93 @@
 
 class execution_summary(luigi.Config):
     summary_length = luigi.IntParameter(default=5)
-    legacy_run_result = luigi.BoolParameter(default=True)
+
+
+class LuigiRetCodes:
+    """
+    All possible return codes for **LuigiRunResult.status** when the argument ``detailed_summary=True`` in
+    *luigi.run() / luigi.build*. Here are the codes and what they mean:
+
+    =============================  =====  ========================================================== 
+    Return Code Name               Value  Meaning 
+    =============================  =====  ========================================================== 
+    SUCCESS                        0      There were no failed tasks or missing dependencies 
+    SUCCESS_WITH_RETRY             1      There were failed tasks but they all succeeded in a retry 
+    FAILED                         2      There were failed tasks
+    FAILED_AND_SCHEDULING_FAILED   3      There were failed tasks and tasks whose scheduling failed
+    SCHEDULING_FAILED              4      There were tasks whose scheduling failed
+    NOT_RUN                        5      There were tasks that were not granted run permission by the scheduler
+    MISSING_EXT                    6      There were missing external dependencies
+    =============================  =====  ==========================================================
```

Comment:
Does this format nicely in python's rst?

Reply:
Ah, I now see your screenshot. It formats really nice. Maybe you even want to point the old documentation that describes the various retcodes to this table?

Reply:
I will point the old documentation to this status code that `LuigiRunResult` exposes 🍸